### PR TITLE
[sailfish-browser] Send updated composition metrics upon VKB activation/deactivation

### PR DESCRIPTION
**Pull request by rojkov**

### CODE REVIEW

File: src/declarativewebcontainer.cpp [hunk 1]
```diff
@@ -897,27 +896,6 @@ void DeclarativeWebContainer::updatePageFocus(bool focus)
...
-    }
-#else
-    // TODO: remove once keyboard height is not zero when hidden and take above #if 0 block into use.
-    vkbHeight = 440;
-    if (width() > height()) {
-        vkbHeight = 340;
-    }
-    vkbHeight *= DeclarativeWebUtils::instance()->silicaPixelRatio();
-#endif
-    m_inputPanelOpenHeight = vkbHeight;
```
Comment (rainemak) [inline src/declarativewebcontainer.cpp]: Remove ```m_inputPanelOpenHeight``` from the header and initialization from the constructor.

**Comment by rainemak** [comment]
Works nicely! Tested with latest embedlite-components (https://github.com/nemomobile-packages/embedlite-components/commit/5659e16c123110554f9b2677d91b97b30ae3eb01). Update embedlite-components requirement to spec before merging.

LGTM with minor cleanup.

**Comment by rojkov** [comment]
Removed m_inputPanelOpenHeight

**Comment by rojkov** [comment]
Updated dep on embedlite-components

**Pull request merged**